src/tokens.ts

```diff
@@ -218,6 +218,16 @@ export const TOKEN_SYMBOLS_MAP = {
     },
     coingeckoId: "havven",
   },
+  SOL: {
+    name: "Solana",
+    symbol: "SOL",
+    decimals: 9,
+    addresses: {
+      [CHAIN_IDs.SOLANA]: "So11111111111111111111111111111111111111112",
```

Comment:
Since we also map ETH to WETH addresses, I guess it makes sense to map SOL to WSOL as well.